packages/inertia-vue/src/plugin.js

```diff
@@ -0,0 +1,13 @@
+import link from './link'
+import remember from './remember'
+import { Inertia } from '@inertiajs/inertia'
+
+export default {
+  instance: null,
+  install(Vue) {
+    Object.defineProperty(Vue.prototype, '$inertia', { get: () => Inertia })
```

Comment:
According to documentation, you can use globalProperties instead of prototype.
https://v3.vuejs.org/api/application-config.html#globalproperties

Reply:
This is actually the Vue 2 plugin (I made some changes to the other adapters while making the Vue 3 plugin).